src/TypeScript/Analyzer/Resources.resx

```diff
@@ -123,6 +123,9 @@
   <data name="ERR_ParsingError_General" xml:space="preserve">
     <value>Failed to parse file '{0}' at line {1}: {2} {3}</value>
   </data>
+  <data name="ERR_ParsingError_LinterNotInitialized" xml:space="preserve">
+    <value>Linter is undefined</value>
```

Comment:
Isn't this a constant value defined by an external component i.e. `eslintbridge`? Do we know if the returned string is localised? (if it is localised, we have a problem since we only handle a single default language)

Assuming it is an external, non-localised fixed value, it would be better to define it is a constant in the code. There's no reason for it to be a resource.